mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -361,13 +359,36 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
         }),
         option: {
           headerStatusBarHeight: 1,
-          headerBackgroundContainerStyle: tailwind('border-b', {
-            'border-gray-200': isLight,
-            'border-gray-700': !isLight,
-            '-top-5': Platform.OS !== 'web'
-          }),
-          header: () => null,
-          headerBackTitleVisible: false
+          headerTitle: '',
+          headerBackTitleVisible: false,
+          header: (): JSX.Element => {
+            return (
+              <ThemedViewV2
+                style={tailwind('flex flex-col px-5 pt-3 pb-5 rounded-t-xl-v2')}
+              >
+                <ThemedTouchableOpacityV2
+                  onPress={() => {
+                    setShowAssetSortBottomSheet(false)
+                    dismissModal(true)
+                  }}
+                  style={tailwind('self-end pt-2.5')}
+                >
+                  <ThemedIcon
+                    dark={tailwind('text-mono-dark-v2-900')}
+                    light={tailwind('text-mono-light-v2-900')}
+                    iconType='Feather'
+                    name='x-circle'
+                    size={20}
+                  />
+                </ThemedTouchableOpacityV2>
+                <ThemedText
+                  style={tailwind('text-xl font-normal-v2 pt-0.5')}
```

Comment:
compensated by the "internal padding" of close icon
```suggestion
                  style={tailwind('text-xl font-normal-v2')}
```